arlas-tests/src/test/java/io/arlas/server/tests/rest/explore/AbstractProjectedTest.java

```diff
@@ -197,20 +203,34 @@ public void testReturnedGeometriesFilter() throws Exception {
     }
 
     @Test
-    public void testReturnedGeometriesFilterWithEmptyColumnFilter() throws Exception {
+    public void testReturnedGeometriesFilterWithNoColumnFilter() throws Exception {
 
         search.returned_geometries = "geo_params.geometry,geo_params.second_geometry";
 
-        handleReturnedMultiGeometries(post(search, ""), search.returned_geometries);
+        handleReturnedMultiGeometries(post(search), search.returned_geometries);
         handleReturnedMultiGeometries(givenFilterableRequestParams().param("include", search.projection.includes)
-                .header(COLUMN_FILTER, "")
                 .param("returned_geometries",  search.returned_geometries)
                 .when().get(getUrlPath("geodata"))
                 .then(), search.returned_geometries);
 
         search.returned_geometries = null;
     }
 
+    @Test
+    public void testReturnedGeometriesFilterWithEmptyColumnFilter() throws Exception {
+
+        search.returned_geometries = "geo_params.geometry,geo_params.second_geometry";
+
+        handleReturnedMultiGeometries(post(search), search.returned_geometries);
```

Comment:
shouldn't this test be also a `handleUnavailableCollection` on a post query ?

Reply:
ok